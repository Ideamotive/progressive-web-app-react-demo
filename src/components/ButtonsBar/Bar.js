import React from 'react';
import styled from 'styled-components';

export default styled.div`
   position: absolute;
    width: 100%;
    height: 100px;
    left: 0;
    bottom: 70px;
    z-index: 3;
    text-align: center;
    background-color: ${props => props.backgroundColor };
  `;
