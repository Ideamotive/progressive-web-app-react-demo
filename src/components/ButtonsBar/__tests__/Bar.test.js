import React from 'react';
import * as renderer from 'react-test-renderer';

import Bar from '../Bar';

describe('<Bar/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(<Bar
        backgroundColor={'#fff'}
      />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

