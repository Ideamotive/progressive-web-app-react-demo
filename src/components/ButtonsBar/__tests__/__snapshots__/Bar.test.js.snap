// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bar/> component should match to snapshot 1`] = `
<div
  className="sc-bdVaJa ciCIem"
/>
`;
